# 數據類型

## Never

手動報錯時使用

```typescript
const throwError = (message: string): never => {
  throw new Error(message)
}

const doSome = (): void => {
  throwError('error')
}
```
